investigations/Possible_Instrument_View_Improvements.md

```diff
@@ -0,0 +1,67 @@
+# Investigation of Possible Instrument 2.0 Effects on Instrument View Performance.
+This document outlines areas in which the performance of the instrument view may benefit from the introduction of the new Instrument 2.0 caching layers. Tests were carried out using the Microsoft Visual Studio debugger and profiling tools in conjunction with `Callgrind` on Linux which provided information on which Instrument 1.0 methods are most heavily used when interacting with the instrument view. The instrumentation was taken for the duration of the creation of the `InstrumentWidget`.
```

Comment:
I would rather that the scope of the investigation was stated to be wider than looking at `Instrument 2.0`. If it turned out that the instrument access was fine and the rate determining step was somewhere in the use of opengl, we would still want to know that, and would still have performed fixes for that.